src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -1601,6 +1601,8 @@ bool CompilerOpenFPGA::Placement() {
       pincommand += " random";
     } else if (PinAssignOpts() == PinAssignOpt::In_Define_Order) {
       pincommand += " in_define_order";
+    } else if (PinAssignOpts() == PinAssignOpt::Free) {
```

Comment:
@alain-rs  please check if this is okay.

Reply:
@taoli4rs we need this change because it was not captured in the previous PR. Does this look ok to you ?